internal/github.com/rackspace/gophercloud/openstack/orchestration/v1/stackevents/requests.go

```diff
@@ -153,8 +153,8 @@ type ListResourceEventsOpts struct {
 	SortDir SortDir `q:"sort_dir"`
 }
 
-// ToResourceEventsListQuery formats a ListOpts into a query string.
-func (opts ListOpts) ToResourceEventsListQuery() (string, error) {
+// ToResourceEventListQuery formats a ListOpts into a query string.
+func (opts ListResourceEventsOpts) ToResourceEventListQuery() (string, error) {
```

Comment:
sorry, no this was a bug I discovered. I will make a PR to add them to gophercloud